{"cve": {"id": "CVE-2025-26269", "sourceIdentifier": "cve@mitre.org", "published": "2025-04-17T18:15:49.073", "lastModified": "2025-04-22T18:15:58.627", "vulnStatus": "Awaiting Analysis", "cveTags": [], "descriptions": [{"lang": "en", "value": "DragonflyDB Dragonfly through 1.28.2 (fixed in 1.29.0) allows authenticated users to cause a denial of service (daemon crash) via a Lua library command that references a large negative integer."}, {"lang": "es", "value": "DragonflyDB Dragonfly hasta la versi\u00f3n 1.28.2 permite a los usuarios autenticados provocar una denegaci\u00f3n de servicio (falla del daemon) a trav\u00e9s de un comando de la librer\u00eda Lua que hace referencia a un entero negativo grande."}], "metrics": {"cvssMetricV31": [{"source": "cve@mitre.org", "type": "Secondary", "cvssData": {"version": "3.1", "vectorString": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L", "baseScore": 3.3, "baseSeverity": "LOW", "attackVector": "LOCAL", "attackComplexity": "LOW", "privilegesRequired": "LOW", "userInteraction": "NONE", "scope": "UNCHANGED", "confidentialityImpact": "NONE", "integrityImpact": "NONE", "availabilityImpact": "LOW"}, "exploitabilityScore": 1.8, "impactScore": 1.4}]}, "weaknesses": [{"source": "cve@mitre.org", "type": "Secondary", "description": [{"lang": "en", "value": "CWE-191"}]}], "references": [{"url": "https://gist.github.com/ankki-zsyang/d8215cf6e868d07546eaa5346a884ebd", "source": "cve@mitre.org"}, {"url": "https://github.com/dragonflydb/dragonfly/commit/4612aec9a78e3f604e6fb19bee51acde89723308", "source": "cve@mitre.org"}, {"url": "https://github.com/dragonflydb/dragonfly/issues/4468", "source": "cve@mitre.org"}]}}